### PR TITLE
Add custom boolean toggle control for water control metadata

### PR DESCRIPTION
**Pull request by gregdingle**
# Description

Following the mocks https://chanzuckerberg.invisionapp.com/share/A6V572ZSBU5#/screens/396544827/comments, this adds a custom control based on semantic UI `Checkbox`. Rendering the control is determined by the new `boolean?` method of `MetadataField`. Currently, only fields that have options "Yes" and "No" are considered boolean.

image

image

I also added an ordering of fields based on the design. 

# Notes

* please ignore the snyk warning because I'm going to merge this branch into another branch, not master
* there was a semantic ui CSS bug such that the default `Checkbox` with toggle is not rendering the circle when unselected
* `force_options` does not seem to do anything in the upload flow. All fields that have options are "forced" by the UI controls.
* only other yes/no field I saw in the codebase was "antibiotics applied"
* I refactored the playground to use `children` as that is more idiomatic

# Tests

* new rspec test of `boolean?`
* load playground, see both states of toggle, see click behavior
* load upload flow, check apply all works

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
 end
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  # config.fixture_path = "#{::Rails.root}/spec/fixtures"
```
Comment (jshoe) [inline spec/rails_helper.rb]: should this be here?

Comment (gregdingle) [reply spec/rails_helper.rb]: yes, I just commented it out like other lines in the file because we are not using rspec fixtures

**Comment by happyimadesignr** [comment]
Hey @gregdingle, does this mean you're ready to share again? I'd like to review it again after you update based on the feedback. cc @oliviabholmes

**Comment by gregdingle** [comment]
@happyimadesignr yes, I can show you it again after the changes we agreed on Tuesday. You can also see all those changes in the screenshots above, except the 200ms animation.

**Comment by happyimadesignr** [comment]
Hey @gregdingle, it looks good, thanks! I didn't see it b/c the IT image thing.

Can you make the gray circle $light-gray, I know you followed the designs, but looking at it again, it would be better a little darker.

Does the gray circle have a gradient on it, if it does, can you remove it, it's hard to tell on the screenshot.

Thanks so much!!

**Pull request merged**